lib/instrumental.js

```diff
@@ -233,6 +237,21 @@ exports.init = function instrumental_init(startup_time, config, events) {
       recordCounterRates = config.instrumental.recordCounterRates;
     }
 
+    // do no filtering by default
+    if(typeof(config.instrumental.metricFilters) === 'undefined'){
+      metricFilters = [];
+    } else {
+      metricFilters = config.instrumental.metricFilters;
```

Comment:
In general, we don't validate any of these configuration options.  I definitely think that, in the future, we should make all of these more careful, but for now, it's up to the user, just like the rest of the things.